Lib/test/test_typing.py

```diff
@@ -239,8 +239,6 @@ def test_cannot_instantiate_vars(self):
             TypeVar('A')()
 
     def test_bound_errors(self):
-        with self.assertRaises(TypeError):
-            TypeVar('X', bound=42)
```

Comment:
`bound=Union` might be a good replacement test case.

Reply:
added